### DISCUSSION TITLE
Welcome to txdx Discussions!

### DISCUSSION THREAD

**Discussion by ariejan** [ariejan/txdx#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by fleapower** [comment]
Just downloaded TXDX.  In the Getting Started section, it says I can name the todo.txt file anything I want, but I don't see an option for that in settings.  What am I missing?  Thanks!

**Reply by ariejan** [reply]
Hey, thanks for pointing that out. Since I created that screen I moved to a directory based approach (ssh, but things like notes and attachments will be added later on). The result is that I expect a `todo.txt` (and `archive.txt`) file right now. If they don't exist, they will be created automatically.

Edit: @fleapower where exactly did you see this message? If it's in the app, are you running the latest version?

**Reply by fleapower** [reply]
Thanks.  It was on this page:  https://www.txdx.eu/getting-started/
